lr1110/Makefile.setup

```diff
@@ -27,12 +27,15 @@ $(LR1110_DIR)/radio_planner.c: $(SEEED_DIR)/smtc_hal/smtc_modem_hal.c
 	cp $(LR1110_DIR)/seeed/lora_basics_modem/smtc_modem_core/radio_planner/src/radio_planner.c $(LR1110_DIR)/radio_planner.c
 	cd $(LR1110_DIR); patch -p0 < ../radio_planner.patch
 
-$(SEEED_DIR)/smtc_hal/smtc_modem_hal.c:
+
+$(SEEED_DIR):
+	mkdir -p $(SEEED_DIR)
+
+$(SEEED_DIR)/smtc_hal/smtc_modem_hal.c: | $(SEEED_DIR)
 	curl -L --output $(ZIP_FILE) "https://codeload.github.com/Seeed-Studio/Seeed_Wio_WM1110_Dev_Board/zip/$(VERSION_HASH)"
-	unzip -q $(ZIP_FILE) -d $(SEEED_DIR)
-	rm -f $(ZIP_FILE) # Remove zip file after unzip
-	mv $(EXTRACTED_DIR)/* $(SEEED_DIR)/
-	rmdir $(EXTRACTED_DIR)  # move unzipped code to SEEED_DIR, remove original directory
+	# Strip-components removes leading directory path (i.e., Seeed_Wio_WM1110_Dev_Board-$(HASH)/) during extraction
+	bsdtar --strip-components=1 -xf $(ZIP_FILE) --cd $(SEEED_DIR)
```

Comment:
Are we sure this is the tool to go with? It's not installed on Ubuntu by default (although it is installable). I'm worried if it's available on other Linux distros. I'm not saying it needs to change, but we should consider it.

Reply:
Ugh, midnight google made that seem like an easy / robust solution, and it happened to already be on my machine so I was hopeful. No, I don't think this is worth installing a new tool for; I'll change this.

Reply:
... to wit, plain `tar` also supports that flag 🙃